src/main/java/edu/kis/powp/jobs2d/command/OperateToCommand.java

```diff
@@ -20,4 +20,15 @@ public void execute(Job2dDriver driver) {
         driver.operateTo(posX, posY);
     }
 
+    @Override
+    public DriverCommand clone(){
+        DriverCommand dc = null;
+        try {
+            dc = (DriverCommand) super.clone();
+        } catch (CloneNotSupportedException e) {
+            e.printStackTrace();
+        }
```

Comment:
Is there a real risk of an exception here? I would resign from catch block here.